airflow/exceptions.py

```diff
@@ -358,16 +362,15 @@ def __init__(
         trigger,
         method_name: str,
         kwargs: dict[str, Any] | None = None,
-        timeout: datetime.timedelta | None = None,
+        trigger_timeout: datetime.datetime | None = None,
+        trigger_timeout_reason: str | None = None,
```

Comment:
this seems to me like breaking backcompat

Reply:
specifically, you are removing the `timeout` param.
why do we need to remove that?
why do we need to add `trigger_` prefix? isn't it implied?

Reply:
I agree, I ignored the possibility that the user might use it in his operators or unit tests. I will replace trigger_timeout and trigger_timeout_reason by timeout and timeout_reason.